xs2a-adapter-app/src/main/java/de/adorsys/xs2a/adapter/config/RestConfiguration.java

```diff
@@ -61,4 +62,9 @@ Psd2AccountInformationService psd2AccountInformationService(AspspReadOnlyReposit
     Pkcs12KeyStore pkcs12KeyStore() throws CertificateException, NoSuchAlgorithmException, KeyStoreException, IOException {
         return new Pkcs12KeyStore(System.getProperty("pkcs12.keyStore"));
     }
+
+    @Bean
+    AspspCsvService aspspCsvService() {
+        return new AspspCsvServiceImpl(new LuceneAspspRepositoryFactory().newLuceneAspspRepository());
+    }
```

Comment:
If you agree with https://github.com/adorsys/xs2a-adapter/pull/182/files#r321748401 and https://github.com/adorsys/xs2a-adapter/pull/182/files#r321749178, I would better change this code to:
```
@Bean
AspspCsvService aspspCsvService(AspspRepository aspspRepository) {
     return new AspspCsvServiceImpl(aspspRepository);
}
```